webpages/js/metaanalyses.js

```diff
@@ -1478,6 +1489,16 @@
       });
     });
 
+    // Add unhide button for the needed papers
```

Comment:
@Jnnybrrn I've try to do this, but it was too dirty. If you want I can continue to look at it